.github/workflows/ci.yml

```
@@ -57,6 +57,8 @@ jobs:
         run: cd packages/foreman_server && mix credo --min-priority higher
       - name: Doctor
         run: cd packages/foreman_server && mix doctor --raise
+      - name: Setup database
+        run: cd packages/foreman_server && MIX_ENV=test mix ecto.create && MIX_ENV=test mix ecto.migrate && MIX_ENV=test mix event_store.init
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2209:warning:1:31: Use var=$(command) to assign output (or quote to assign string)

(shellcheck)